### PR TITLE
fix: update transaction service endpoints

### PR DESCRIPTION
**Pull request by ib-tycho**

### CODE REVIEW

File: ape_safe.py [hunk 1]
```diff
@@ -34,15 +34,15 @@
     1: 'https://safe-transaction.mainnet.gnosis.io',
```
Comment (mjdatadog) [inline ape_safe.py:34]:
```suggestion
    1: 'https://safe-transaction-mainnet.safe.global',
```
All networks updates have completed as per the schedule [here](https://forum.gnosis-safe.io/t/transaction-service-migration-october-2022/1550)

File: ape_safe.py [hunk 2]
```diff
@@ -34,15 +34,15 @@
...
     4: 'https://safe-transaction.rinkeby.gnosis.io',
     5: 'https://safe-transaction.goerli.gnosis.io',
-    10: 'https://safe-transaction.optimism.gnosis.io',
+    10: 'https://safe-transaction-optimism.safe.global',
     56: 'https://safe-transaction.bsc.gnosis.io',
     100: 'https://safe-transaction.xdai.gnosis.io',
```
Comment (mjdatadog) [inline ape_safe.py:39]:
```suggestion
    100: 'https://safe-transaction-gnosis-chain.safe.global',
```
All networks updates have completed as per the schedule [here](https://forum.gnosis-safe.io/t/transaction-service-migration-october-2022/1550)

File: ape_safe.py [hunk 3]
```diff
@@ -34,15 +34,15 @@
...
     137: 'https://safe-transaction.polygon.gnosis.io',
```
Comment (mjdatadog) [inline ape_safe.py:40]:
```suggestion
    137: 'https://safe-transaction-polygon.safe.global',
```
All networks updates have completed as per the schedule [here](https://forum.gnosis-safe.io/t/transaction-service-migration-october-2022/1550)

File: ape_safe.py [hunk 4]
```diff
@@ -34,15 +34,15 @@
     1: 'https://safe-transaction.mainnet.gnosis.io',
     4: 'https://safe-transaction.rinkeby.gnosis.io',
     5: 'https://safe-transaction.goerli.gnosis.io',
-    10: 'https://safe-transaction.optimism.gnosis.io',
+    10: 'https://safe-transaction-optimism.safe.global',
     56: 'https://safe-transaction.bsc.gnosis.io',
```
Comment (mjdatadog) [inline ape_safe.py:38]:
```suggestion
    1: 'https://safe-transaction-bsc.safe.global,
```
All networks updates have completed as per the schedule [here](https://forum.gnosis-safe.io/t/transaction-service-migration-october-2022/1550)

**Review by mjdatadog — COMMENTED** [review]
All networks updates have completed as per the schedule [here](https://forum.gnosis-safe.io/t/transaction-service-migration-october-2022/1550)

**Review by banteg — APPROVED** [review]

**Pull request merged**